auth/client/src/main/resources/schema/encrypted-expression-1_0.xsd

```diff
@@ -0,0 +1,240 @@
+<?xml version="1.0" encoding="UTF-8"?>
+    <!--
+      ~ JBoss, Home of Professional Open Source.
+      ~ Copyright 2023 Red Hat, Inc., and individual contributors
+      ~ as indicated by the @author tags.
+      ~
+      ~ Licensed under the Apache License, Version 2.0 (the "License");
+      ~ you may not use this file except in compliance with the License.
+      ~ You may obtain a copy of the License at
+      ~
+      ~     http://www.apache.org/licenses/LICENSE-2.0
+      ~
+      ~ Unless required by applicable law or agreed to in writing, software
+      ~ distributed under the License is distributed on an "AS IS" BASIS,
+      ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+      ~ See the License for the specific language governing permissions and
+      ~ limitations under the License.
+    -->
+
+<xsd:schema xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xss="http://www.w3.org/2001/XMLSchema"
+            targetNamespace="urn:encrypted:expression:1.0"
+            xmlns="urn:encrypted:expression:1.0"
+            elementFormDefault="qualified"
+            attributeFormDefault="unqualified"
+            version="1.0">
+
+    <!-- Encrypted Expression elements -->
+    <xsd:element name="encrypted-expression" type="encrypted-expression-type"/>
+
+    <xsd:complexType name="encrypted-expression-type">
+        <xsd:annotation>
+            <xsd:documentation>
+                Configuration for encrypted expressions and expression-resolvers to be used across different client configurations.
+            </xsd:documentation>
+        </xsd:annotation>
+        <xsd:all>
+            <xsd:element name="credential-stores" type="credential-stores-type" minOccurs="1" maxOccurs="1"/>
+            <xsd:element name="expression-resolvers" type="expression-resolvers-type" minOccurs="1" maxOccurs="1"/>
+        </xsd:all>
+    </xsd:complexType>
+
+    <!-- Credential Stores elements -->
+    <xsd:complexType name="credential-stores-type">
+        <xsd:annotation>
+            <xsd:documentation>
+                Complex type to contain the definitions of all the credential stores used to back the expression-resolvers configured.
+            </xsd:documentation>
+        </xsd:annotation>
+        <xsd:sequence>
+            <xsd:element name="credential-store" type="credential-store-type" minOccurs="0" />
+        </xsd:sequence>
+    </xsd:complexType>
+
+    <xsd:complexType name="credential-store-type">
```

Comment:
@PrarthonaPaul Just wondering, is it possible for this new schema to reference the types from the existing elytron-client xsd schema? With some include?

Reply:
Are there any examples of this? 
If we can reuse credential-store-type from the elytron schema, it would make it much simpler.

Reply:
@PrarthonaPaul I do not know of any examples, but maybe it is possible to do https://stackoverflow.com/questions/8194112/basics-of-referencing-a-xsd-schema-from-another-schema ? It needs some investigations of the schema possibilities

Reply:
I have added a reference to the elytron schema using the method mentioned in the post you shared. Thanks!